README.md

```diff
@@ -1304,8 +1304,26 @@ character_portrait.huge # => "http://image.eveonline.com/Character/90729314_512.
 #### Industry
 
 ##### List character industry jobs
+```ruby
+options = { token: 'token123', character_id: 90729314 }
+
```

Comment:
Documentation also need some love :)

Reply:
Done ;-)
I love the way you code...

Perhaps I should rename the names of files too ?

Reply:
Rename files too. And make documentation looks like everywhere in project. E.g. https://github.com/biow0lf/eve_online#get-character-assets